sound/soc/sof/imx/common.c

```diff
@@ -0,0 +1,75 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// This file is provided under a dual BSD/GPLv2 license.  When using or
+// redistributing this file, you may do so under either license.
+//
+// Debug support for audio DSP on i.MX8
+
+#include <sound/sof/xtensa.h>
```

Comment:
Lets not call this 'debug support'. This file will be used for more than that. We could say:

```Common helpers for audio DSP on i.MX```